### PR TITLE
feat: add mapping to verzoek plugin

### PR DESCRIPTION
**Pull request by mritense**
Seems to be complete, only need input for valid examples of paths which can be put in the mapping fields. Putting on WIP for now. Closes https://ritense.tpondemand.com/entity/47744-fe-verzoek-plugin-mapping-verzoek-object

### CODE REVIEW

File: projects/valtimo/plugin/src/lib/plugins/verzoek/components/verzoek-configuration/verzoek-configuration.component.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline projects/valtimo/plugin/src/lib/plugins/verzoek/components/verzoek-configuration/verzoek-configuration.component.scss:2]:
```suggestion
 * Copyright 2015-2023 Ritense BV, the Netherlands.
```

File: projects/valtimo/user-interface/src/lib/components/radio/radio.component.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<!--
+  ~ Copyright 2015-2020 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline projects/valtimo/user-interface/src/lib/components/radio/radio.component.html:2]:
```suggestion
  ~ Copyright 2015-2023 Ritense BV, the Netherlands.
```

File: projects/valtimo/user-interface/src/lib/components/radio/radio.component.scss [hunk 3]
```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline projects/valtimo/user-interface/src/lib/components/radio/radio.component.scss:2]:
```suggestion
 * Copyright 2015-2023 Ritense BV, the Netherlands.
```

File: projects/valtimo/user-interface/src/lib/components/radio/radio.component.ts [hunk 4]
```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline projects/valtimo/user-interface/src/lib/components/radio/radio.component.ts:2]:
```suggestion
 * Copyright 2015-2023 Ritense BV, the Netherlands.
```

File: projects/valtimo/user-interface/src/lib/components/radio/radio.module.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline projects/valtimo/user-interface/src/lib/components/radio/radio.module.ts:2]:
```suggestion
 * Copyright 2015-2023 Ritense BV, the Netherlands.
```

File: projects/valtimo/user-interface/src/lib/models/radio.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline projects/valtimo/user-interface/src/lib/models/radio.ts:2]:
```suggestion
 * Copyright 2015-2023 Ritense BV, the Netherlands.
```

File: gradlew [hunk 7]
```diff
@@ -1,16 +1,16 @@
 #!/bin/sh
 
 #
-# Copyright © 2015-2021 the original authors.
+# Copyright 2015-2023 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline gradlew:4]: We don't have copyright on this file

File: cypress/support/e2e.ts [hunk 8]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline cypress/support/e2e.ts:2]: I doubt we can claim copyright on a sample file

File: cypress/support/commands.ts [hunk 9]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline cypress/support/commands.ts:2]: I doubt we can claim copyright on a sample file

File: e2e/cypress/support/commands.ts [hunk 10]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline e2e/cypress/support/commands.ts:2]: I doubt we can claim copyright on a sample file

File: e2e/cypress/support/e2e.ts [hunk 11]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
```
Comment (theo-ritense) [inline e2e/cypress/support/e2e.ts:2]: I doubt we can claim copyright on a sample file

**Review by marijnritense — DISMISSED** [review]

**Pull request merged**